js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -198,6 +251,25 @@ module.exports = function (context) {
         }
         break;
       }
+    },
+    Identifier: function Identifier(path, state) {
```

Comment:
These are from https://github.com/substack/insert-module-globals/blob/769f4bd08e01b77a620f2e2de1d2249c8311b0a2/index.js#L25-L51, which is the list of globals supported by browserify. 

`__dirname` and `__filename` would be totally useless in the browser, except *maybe* for debugging but sourcemaps would be better in that case.